pkgs/by-name/gz/gz-math/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  eigen,
+  fetchFromGitHub,
+  gz-cmake,
+  gz-utils,
+  python3Packages,
+  ruby,
+  swig,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gz-math";
+  version = "8.2.0";
+
+  src = fetchFromGitHub {
+    owner = "gazebosim";
+    repo = "gz-math";
+    tag = "gz-math${lib.versions.major finalAttrs.version}_${finalAttrs.version}";
+    hash = "sha256-UpwgQrQrFuBe/ls9HtZy+UgO8b2ObHLCmCS6epEwOPo=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gz-cmake
+  ];
+
+  buildInputs = [
+    gz-utils
+    eigen
+    ruby
+    swig
+  ];
+
+  doCheck = true;
+
+  nativeCheckInputs = [
+    python3Packages.pybind11
```

Comment:
I doubt you meant to add this to `nativeCheckInputs`. I cannot tell if you are interested in the Python bindings or not.